gc/base/EnvironmentBase.hpp

```diff
@@ -664,8 +693,9 @@ class MM_EnvironmentBase : public MM_BaseVirtual
 		,_currentTask(NULL)
 		,_slaveThreadCpuTimeNanos(0)
 		,_freeEntrySizeClassStats()
-		,_oolTraceAllocationBytes(0)
-		,_traceAllocationBytes(0)
+		,_oolTraceAllocationBytesForTracepoint(0)
```

Comment:
Now that I realized that Hook and Tracepoint reside on OpenJ9 side, I'm wondering if it makes sense to expand these names with ... ForTracepoint/ForHook. They make little sense to someone how looks OMR code and is not aware of OpenJ9 code. @dmitripivkine what do you think?

Reply:
I agree. "ForTracepoint/ForHook" does not explain what these variables are used for without understanding how "Tracepoint" or "Hook" are used for

Reply:
GCExtensionsBase.hpp:
oolObjectSamplingBytesGranularityForTracepoint --> oolObjectSamplingBytesGranularity
objectSamplingBytesGranularityForHook --> objectSamplingBytesGranularity
EnvironmentBase.hpp:
_oolTraceAllocationBytesForTracepoint --> _oolTraceAllocationBytes
_traceAllocationBytesForHook --> _traceAllocationBytes

Reply:
Yeah, let's restore the naming back. Sorry, I know I proposed the change to start with, since I did not like one just being OOL and the other not. (and I still don't like that separation, since both of them end up doing OOL)

Perhaps one should be coarseSamplingAllocatedBytes/coarseSamplingByteGranularity (what OOL is now) and the other one is fineSamplingAllocatedByte/fineSamplingByteGranularity , but unless Dmitri strongly agrees that OOL differentiation is confusing, leave it as you suggested.

Reply:
I am ok to reuse old names. At least it simplifies merge for this change.